powerlevel9k.zsh-theme

```diff
@@ -1317,8 +1317,8 @@ prompt_powerlevel9k_setup() {
 
   # Display a warning if the terminal does not support 256 colors
   local term_colors
-  term_colors=$(echotc Co)
-  if (( $term_colors < 256 )); then
+  term_colors=$(echotc Co 2>/dev/null)
+  if (( ! $? && ${term_colors:-0} < 256 )); then
```

Comment:
It certainly took me a few rounds to get the `$?` test right. If that confused me *and* you, it will probably confuse other people too. Perhaps the most readable (maintainable) approach really is to use `${term_colors:-256}` and get rid of the `$?` check entirely.